wgpu-core/src/command/render.rs

```diff
@@ -2517,6 +2546,36 @@ fn multi_draw_indirect(
         ),
     );
 
+    let (indirect_raw, offset) = if count == 1 && state.device.indirect_draw_validation.is_some() {
```

Comment:
Would there be significant lift to dealing with MDI by doing these calls in a loop?

Reply:
I was thinking we would need a fastpath for MDI but maybe we don't (or can add it later). I will add the loop.

Reply:
Ah, no, this doesn't work; unless we call `state.raw_encoder.draw_indirect/draw_indexed_indirect` in the loop as well since the destination buffer might change between calls to `IndirectDrawValidationBatcher.add()`.

Reply:
Or we could keep track of the destination buffer and only call `draw_indirect/draw_indexed_indirect` once it changes or at the end of the loop.

Reply:
I pushed this as a separate commit.